relay-server/src/services/processor.rs

```diff
@@ -680,6 +700,12 @@ impl ProcessingExtractedMetrics {
 
                 if reset_extracted_from_indexed.contains(&namespace) {
                     bucket.metadata.extracted_from_indexed = false;
+                    // If this metric is a usage metric, and we dropped the indexed payload due to
+                    // rate limiting, we want to remove the tag, so that the amount of
+                    // transactions/spans can still be counted correctly.
+                    if BucketInspector(bucket).is_usage_metric() {
+                        bucket.tags.remove("extracted_from_indexed");
```

Comment:
As discussed we want to name this `indexed` and probably should have a const for the tag name.